packages/os/prairiedog-toml

```diff
@@ -0,0 +1,22 @@
+[required-extensions]
+boot = "v1"
++++
+{{#if settings.boot}}
+{{#if settings.boot.reboot-to-reconcile}}
+reboot-to-reconcile = {{settings.boot.reboot-to-reconcile}}
+{{/if}}
+
+{{#if settings.boot.kernel}}
+[kernel]
+{{#each settings.boot.kernel}}
+"{{@key}}" = [ {{#each this}}"{{this}}",{{/each}} ]
+{{/each}}
+{{/if}}
+
+{{#if settings.boot.init}}
+[init]
+{{#each settings.boot.init}}
+"{{@key}}" = [ {{#each this}}"{{this}}",{{/each}} ]
+{{/each}}
+{{/if}}
+{{/if}}
```

Comment:
nit: can you structure templates like these so they render a completely empty file if there are no corresponding settings, and (ideally) with the correct spacing between the logical sections?

Whitespace issues like that are among my pet peeves when dealing with the output of templating systems. I'd like the output to match what I'd write if I created the file by hand, and as you may have noticed I have feelings about missing or extra newlines.